itou/users/models.py

```diff
@@ -1490,6 +1490,39 @@ def readonly_pii_fields(self):
         return blocked_fields
 
 
+class NirModificationRequest(models.Model):
+    jobseeker_profile = models.ForeignKey(
+        JobSeekerProfile,
+        verbose_name="profil demandeur d'emploi",
+        related_name="+",
+        on_delete=models.CASCADE,
+    )
+    new_nir = models.CharField(verbose_name="nouveau NIR", max_length=15, validators=[validate_nir])
+    job_application = models.ForeignKey(
+        "job_applications.JobApplication",
+        verbose_name="candidature",
+        related_name="+",
+        null=True,
+        on_delete=models.SET_NULL,
+    )
+    requestor = models.ForeignKey(
```

Comment:
```suggestion
    requested_by = models.ForeignKey(
```

C'est plutôt la tendance ;)